web3.js

```diff
@@ -36,6 +36,19 @@ function load(opts) {
   return new Web3(provider(opts.provider))
 }
 
+/**
+ * Returns the provider(s) based on .ararc network id.
+ * @param  {Object} web3
+ * @return {String|Array}
+ */
+function getProvider(web3) {
+  const { network_id } = web3
+  if (network_id && web3[network_id]) {
+    return web3[network_id].providers
```

Comment:
Hmm why use providers instead of provider? We should keep the naming consistent. ethprovider accepts strings and arrays as input

Reply:
I had that thought too but thought it made more sense for an array variable name to be plural. I'd recommend changing `web3.provider` to be `web3.providers` and anyone consuming can provide an array there but it would break backwards compatibility. Pushed the change.